compiler/rustc_codegen_llvm/src/common.rs

```diff
@@ -279,7 +274,7 @@ impl<'ll, 'tcx> ConstMethods<'tcx> for CodegenCx<'ll, 'tcx> {
                 let llval = unsafe {
                     llvm::LLVMRustConstInBoundsGEP2(
                         self.type_i8(),
-                        self.const_bitcast(base_addr, self.type_i8p_ext(base_addr_space)),
+                        self.const_bitcast(base_addr, self.type_ptr_in(base_addr_space)),
```

Comment:
Note to self: `type_ptr_in` doesn't actually make sense, use something like `type_ptr_from` when bringing this back